### ISSUE TITLE
Is it time for end-of-life?

### ISSUE THREAD

**Issue by nex3** [sass/node-sass#3424]
Node Sass hasn't had a release since May 2023. Issues are going unanswered, and this repo is causing at least some amount of user confusion (see https://github.com/sass/node-sass/issues/3421, which is a question about Dart Sass). Installation [appears to be broken](https://github.com/sass/node-sass/issues/3420) for users with recent Python versions. LibSass put out a release eight months ago that still hasn't made it into a Node Sass release.

This all suggests to me that we can no longer credibly describe Node Sass as being "maintained indefinitely" even on a best-effort basis, and that we should officially mark it as end-of-life, archive the repository, and update the npm package to say "END OF LIFE" in big red letters.

@nschonni @xzyfer

**Comment by xzyfer** [comment]
Agreed. The release schedule roughly lines up with major Node releases but I have fallen behind on Node 22. We haven't been adding features in node-sass or keeping up libsass.

**Comment by ntkme** [comment]
Also https://docs.npmjs.com/deprecating-and-undeprecating-packages-or-package-versions
